zephyr/src/supp_main.c

```diff
@@ -154,6 +155,8 @@ static int z_wpas_add_interface(const char* ifname)
 		wpa_printf(MSG_ERROR, "Failed to add iface: %s", ifname);
 		return -1;
 	}
+
+	send_wifi_state_event(ifname, NET_EVENT_WPA_SUPP_CMD_IFACE_ADDED);
```

Comment:
This should be after control interface init to avoid command failures.

Reply:
Done